release/models/bfd/openconfig-bfd.yang

```diff
@@ -508,6 +515,99 @@ module openconfig-bfd {
     uses bfd-session-microbfd-common;
   }
 
+  grouping bfd-interface-session-config {
+    description
+      "Configuration parameters for a static BFD single-hop session on an interface.";
+
+    leaf local-address {
+      type oc-inet:ip-address;
+      description
+        "The local IP address used by the system for the single-hop BFD session
+        specified.";
+    }
+
+    leaf remote-address {
+      type oc-inet:ip-address;
+      description
+        "The remote IP destination that should be used by the system for
+        the single-hop BFD session specified.";
+    }
+
+    leaf admin-down {
```

Comment:
We've generally used `enabled` as this leaf's name -- and have zero called `admin-down`, so let's go for some consistency here and use `enabled`.

Reply:
This was also raised some time ago whether or not this should have been corrected for those that veered but current stats look like:

```
$ find . -name "*.yang" | xargs grep -E "leaf enable(d?) {" | awk '{print $3}' | sort | uniq -c
     15 enable
     51 enabled
```

Reply:
@Pull-eckermann please rename this leaf to `enabled`